events/events.go

```diff
@@ -90,6 +136,25 @@ func EmitPoetWaitRound(nodeID types.NodeID, current, publish types.EpochID, wait
 	)
 }
 
+func EmitWaitingForPoETRegistrationWindow(nodeID types.NodeID, current, publish types.EpochID, wait time.Time) {
+	const help = "Node is waiting for PoET registration window in current epoch to open. " +
+		"After this it will submit challenge and wait until PoET round ends in publish epoch."
+	emitUserEvent(
+		help,
+		false,
+		&pb.Event_WaitingForPoetRegistrationWindow{
+			WaitingForPoetRegistrationWindow: &pb.EventWaitingForPoETRegistrationWindow{
+				Current: current.Uint32(),
+				Publish: publish.Uint32(),
```

Comment:
NIT: `current` epoch is arguably not that interesting, but the `publish` epoch of the ATX and maybe the `target` epoch (always `publish` epoch + 1)

Reply:
fyi: it's old `EmitPoetWaitProof` event only renamed. 
Changing these fields may require more work to adapt any automation which is already dependent on these events.

Maybe someone could spent quarter of an hour to look on all events and what could be improved, so i could include all these changes in this PR?
@poszu @ivan4th @fasmat  WDYT?

Reply:
At the moment these equalities always hold true, since they is hardcoded:

```
current + 1 = publish
publish + 1 = target
```

However the first line doesn't need to be true - theoretically a PoET can run for longer than one epoch, the protocol would allow it. Only the second line is given by the protocol; the target epoch is always one after the publish epoch.

To know the current epoch one can just calculate it based on when that event was emitted and the time of genesis, but the publish epoch is the interesting data - in which epoch are we planning to publish the ATX we are building (and arguably which epoch are we targeting with it).

I'm not insisting on that change and since we don't support PoETs longer than one epoch yet anyway 🙂